app/controllers/placements/schools/mentors_controller.rb

```diff
@@ -31,7 +31,7 @@ def destroy
   private
 
   def set_mentor
-    @mentor = @school.mentors.find(params.require(:id))
+    @mentor = policy_scope(@school.mentors).find(params.require(:id))
   end
```

Comment:
How about following the same pattern here?

```ruby
class MentorsController
  def index
    @page, @mentors = page(mentors.order_by_full_name)
  end
  
  private
  
  def mentors
    policy_scope(@school.mentors)
  end
  
  def set_mentor
    @mentor = mentors.find(params.require(:id))
  end
end
```

I quite like this approach because it defines the scope for this entire controller in a single place. That should make it easier to 'get it right' if we add more actions to this controller in future – we're nudging future developers towards the [pit of success](https://blog.codinghorror.com/falling-into-the-pit-of-success/).